src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { PostPrepared } from '../../types/postsPrepared';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+interface Props {
+  post: PostPrepared,
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    userId,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && <UserInfo key={userId} user={(user)} />}
```

Comment:
No need of `key` here. Keys should be given to the elements inside the array to give the elements a stable identity